app/assets/stylesheets/tailwind_overrides.css

```diff
@@ -86,10 +86,5 @@
     select {
       @apply border;
     }
-
-    input,
-    textarea {
-      background-color: revert;
-    }
```

Comment:
This was added in https://github.com/davidrunger/david_runger/pull/6495 . However, I think that Tailwind's default -- a transparent background -- actually is pretty good, in most cases. On a dark background, we want the textarea to also be dark, not white (which is what `revert` achieves).

In other words, on closer reflection, it turns out that this is just something that we want for the groceries app, because it doesn't have dark mode and because it has an image background.

So, this change moves it to the (unscoped) style tag in `Groceries.vue`.